src/system/sdl/main.c

```diff
@@ -1617,9 +1617,11 @@ static void emsGpuTick()
 
 static s32 start(s32 argc, char **argv, const char* folder)
 {
-    SDL_Init(SDL_INIT_VIDEO | SDL_INIT_AUDIO | SDL_INIT_JOYSTICK);
```

Comment:
We have to initialize SDL first here to detect the keyboard layout for the gamepad mapping we need in `studioInit()`. Pls revert or find a better way to init the config, because it breaks keyboard input for the gamepad.

Reply:
> but why do we need SDL to just load the config

Because `config` calls `tic_sys_default_mapping()` when initializing.

Reply:
Right, that just copies static default enumerable values into `&config->data.options.mapping`... I'm still not seeing why we need SDL init *at that point*...?  Config doesn't even know about SDL (one of many platforms) so I'm not sure I"m understanding the dependency you think is here?

Reply:
_Oh wait, looking at the wrong source file._ *looking closer*